### ISSUE TITLE
emb-m-09 - Sidebar

### ISSUE THREAD

**Issue by bertramlyons** [LibraryOfCongress/embARC#26]
emb-m-09	Sidebar	Yes	2-High	

"Pop-out" buttons are not visible to screen reader users, and do not receive keyboard focus. They can only be activated using a mouse.	

Ensure these buttons are correctly implemented as interactive elements, receive keyboard focus and are appropriately labeled.

**Comment by bertramlyons** [comment]
In progress. Pop out buttons screen reader & keyboard focus.

**Comment by bertramlyons** [comment]
Completed and merged in version 1.3.0.
